src/http-servers/echo-server.js

```diff
@@ -0,0 +1,3 @@
+require('http')
+    .createServer((req, res) =>  req.pipe(res))
+    .listen(8080);
```

Comment:
Use different ports for every server

Reply:
Also, such sort of things are usually put into config which is served by tools like https://www.npmjs.com/package/nconf